Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -162,9 +166,32 @@ public void setSettings(ArrayList<SettingsItem> settings)
 
   public void onBooleanClick(CheckBoxSetting item, int position, boolean checked)
   {
+    SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(mContext);
+    final SharedPreferences.Editor editor = preferences.edit();
+
+    boolean wiimoteOrientation =
+            item.getSetting().getSection().equals(SettingsFile.KEY_WIIMOTE_PLAYER_1) &&
+                    item.getSetting().getKey().equals(SettingsFile.KEY_WIIMOTE_ORIENTATION);
+    boolean customGameSettingsWiimoteOrientation =
+            item.getSetting().getKey().equals(SettingsFile.KEY_WIIMOTE_ORIENTATION + 0);
```

Comment:
Should be fixed but worth another check since I haven't written much code for the new config system.

Reply:
Why is the orientation setting reading from `MAIN_WII_OVERLAY_DISABLES_CONTROLLERS` in the new code? I was assuming you would want to do something with `SettingsFile.KEY_WIIMOTE_ORIENTATION` instead.

Also, the implementation of `setBoolean` is only writing to the SharedPreferences setting and not to any INI file setting, unlike the old code.

Reply:
Oops, I previously picked the wrong setting. This time I picked the right one but was this what you had in mind? It still uses hacky code but is at least contained in one place.

And I'll look into my changes more later but `setBoolean` changes are reflected in the INI with what I've currently written, possibly due to code that follows it. My changes to `SettingsFile.saveCustomGameSettings` no longer trigger but I appear to still get the result I want.

Reply:
This looks more like what I had in mind, yes. I'll leave further comments on the code directly instead of here.

> And I'll look into my changes more later but `setBoolean` changes are reflected in the INI with what I've currently written, possibly due to code that follows it.

I don't see how that is a problem. Did you miss a "not"?

Reply:
Oh good, because that was a pain to write lol.
And I was confused by
> Also, the implementation of setBoolean is only writing to the SharedPreferences setting and not to any INI file setting, unlike the old code.

...but I used `setBoolean` while properly saving the values to the INIs. There isn't a problem with the current code as far as I can tell.

Reply:
I think I'm a little confused now... But if the current code is working, it's all in order.